.github/workflows/pull_request.yml

```diff
@@ -14,6 +14,13 @@ jobs:
       - name: catkin build
         shell: bash
         run: |
+            git submodule update --init --recursive
             source /opt/ros/noetic/setup.bash
             cd catkin_ws
             catkin build
+      - name: simple test
```

Comment:
can we rename the test from "simple test" to something like "sim test" or "planner test" or something so its clear when it fails what its testing

Reply:
fixed